source/_components/growatt_server.markdown

```diff
@@ -0,0 +1,64 @@
+---
+title: "Growatt server PV Inverter Sensor"
+description: "Instructions on how to integrate your growatt server solar inverter within Home Assistant."
+logo: growatt.png
+ha_category:
+  - Sensor
+  - Energy
+ha_release: 0.98
+ha_iot_class: Cloud Polling
+---
+
+This is a sensor to collect information from your growatt inverters using growatt server.
+
+This will log into your growatt account and grab the first "Plant" after which it collects the inverters on this plant and creates sensors for these inverters as well as a total sensor.
```

Comment:
```suggestion
This will log into your Growatt account and grab the first "Plant", after which it collects the inverters on this plant and creates sensors for these inverters as well as a total sensor.
```